doc/api/cli.md

```diff
@@ -1980,6 +1980,21 @@ On a machine with 2 GB of memory, consider setting this to
 $ node --max-old-space-size=1536 index.js
 ```
 
+### `--max_semi_space_size=SIZE` (in megabytes)
+
+Sets the maximum
+[semi-space](https://www.memorymanagement.org/glossary/s.html#semi.space)
+size for V8's [scavenge](https://v8.dev/blog/orinoco-parallel-scavenger)
+garbage collector in MiB.
+Increasing the max size of a semi-space may bring
+throughput improvement for Node.js and the cost is more
+memory consumption (see [#42511](https://github.com/nodejs/node/issues/42511)).
+
+_The default value is
+16MiB for 64-bit systems and 8MiB for 32-bit systems.
+The recommended value is 64MiB or 128MiB if your system has enough
```

Comment:
I'm wondering if there a rational we can include for choosing these as the recommendation?

Reply:
Actually, there isn't too much optional value for the 'max_semi_space_size', because V8 did the [round up](https://chromium.googlesource.com/v8/v8.git/+/refs/heads/main/src/heap/heap.cc#5156) operation to make sure the 'max_semi_space_size' is the power of 2. So, only if we set the value of 'max_semi_space_size' to 16MiB, 32MiB, 64MiB, 128MiB, 256MiB and so on, the behavior is what we expected. I tuned these value on WebTooling and Ghost.js workloads, the result shows that 128MiB is the optimal configuration for both of these workloads.
![result](https://user-images.githubusercontent.com/88813722/160621733-140758a1-b172-4549-a181-e83fa7a79747.png)
I don't know if it is appropriate to put this image into the document. So I just put the link of issue [#42511](https://github.com/nodejs/node/issues/42511) in the document.

Reply:
I'd refrain from recommending specific values, just that higher values can give better throughput but the reader should benchmark that.

It should also mention that the size is effectively tripled because there is more than one semi space. Observe:
```
$ for MB in 16 32 64 128; do
    node --max_semi_space_size=$MB --max_old_space_size=16 -p \
    'const MB = 1<<20; (v8.getHeapStatistics().heap_size_limit - 16*MB) / MB';
  done
48
96
192
384
```

Reply:
@bnoordhuis Thanks for your suggestion! I have updated the patch to guide users to choose the max_semi_space_size for themselves.